@pauliescanlon/gatsby-mdx-embed/src/components/Twitter/Tweet.tsx

```diff
@@ -9,14 +9,14 @@ export interface ITweetProps {
   align?: 'left' | 'center' | 'right'
 }
 
-export const Tweet: FunctionComponent<ITweetProps> = ({
+export const Tweet: FunctionComponent<Props> = ({
   tweetLink,
   theme = 'light',
   align = 'left'
-}: ITweetProps) => (
+}) => (
   <div className="twitter-tweet-mdx-embed" style={{ overflow: 'auto' }}>
     <blockquote className="twitter-tweet" data-theme={theme} data-align={align}>
-      <a href={`https://twitter.com/${tweetLink}?ref_src=twsrc%5Etfw`}>
```

Comment:
i might need to double check this but i'm sure if i didn't have the `ref` in the url i was seeing errors?

Reply:
I removed it in my [`gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder) plugin, because that's used for tracking and I don't want to have tracking enabled.

https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/3fd14921c5c61ae4f48e317b506be6252ed0ea9b/src/transformers/Twitter.js#L19

Reply:
Ah ok. That's probably a good idea!